### PR TITLE
Issue14 readme enchacement

### PR DESCRIPTION
**Pull request by Dongata**
## Background
The other readme was a little bit lean, so we put it some love
  
## Changes done
* Adds prerequisites and where to get them
* Adds references links
* Adds a getting started section
* Adds a next steps guide
  
## Pending to be done
* Would be great to add a 'add a database engine' section under next steps
* Would be great to add a 'how to deploy' section under next steps
 
## Demo

[How it ended](https://github.com/MakingSense/WebApiCore-Seed/tree/issue14-readme-enchacement)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,17 +1,166 @@
...
-* Server Side
-    * Asp.Net Web Api Core
-    * Entity Framework(as DAL)
-    * Net Core DI
-    * Xunit
+# WebapiCore-seed [![Build Status](https://travis-ci.org/MakingSense/WebApiCore-Seed.svg?branch=master)](https://travis-ci.org/MakingSense/WebApiCore-Seed)
+
+## Prerequisites
+* .Net core sdk
+* A text processor / gui
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: done c:

File: Readme.md [hunk 2]
```diff
@@ -1,17 +1,166 @@
...
-## Technologies
-* Server Side
-    * Asp.Net Web Api Core
-    * Entity Framework(as DAL)
-    * Net Core DI
-    * Xunit
+# WebapiCore-seed [![Build Status](https://travis-ci.org/MakingSense/WebApiCore-Seed.svg?branch=master)](https://travis-ci.org/MakingSense/WebApiCore-Seed)
+
+## Prerequisites
+* .Net core sdk
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Done!

File: Readme.md [hunk 3]
```diff
@@ -1,17 +1,166 @@
...
+* A text processor / gui
+
+### Download them
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Done

File: Readme.md [hunk 4]
```diff
@@ -1,17 +1,166 @@
...
+* [For Windows](https://www.microsoft.com/net/download/windows/build)
+* [For Linux](https://www.microsoft.com/net/download/linux/build)
+* [For Mac](https://www.microsoft.com/net/download/macos/build)
+
+## Getting started
+
+* ### For windows users with visual studio
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Branding updated

[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: and removed the bullet points

File: Readme.md [hunk 5]
```diff
@@ -1,17 +1,166 @@
...
+        /// </summary>
+        [HttpGet("id")]
+        [ProducesResponseType(typeof(NewValue), 200)]
+        [ProducesResponseType(typeof(ErrorDto), 400)]
+        [ProducesResponseType(typeof(ErrorDto), 500)]
+        public Task<IActionResult> Get(Guid id){...}
+        ```
+
+    10. Finally it's time to test, for this you need to create two test files, one in `WebApiCoreSeed.UnitTests/Services` to test the service, in this case the file should be called `NewValueServiceTest.cs`, and other in `WebApiCoreSeed.UnitTests/Api/Controllers` for the controller called `NewValueControllerTest.cs`.
+
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Removed the whole section, you're right, a readme is not the place for a mini tutorial

File: Readme.md [hunk 6]
```diff
@@ -1,17 +1,166 @@
...
+## What uses
+
+| Technology  | Description |
+| :---------: | ----------- |
+| [Asp .Net Web Api Core](https://docs.microsoft.com/en-us/aspnet/core/?view=aspnetcore-2.0) | Core framework |
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Done!

File: Readme.md [hunk 7]
```diff
@@ -1,17 +1,166 @@
...
+| [Entity Framework](https://docs.microsoft.com/en-us/ef/core/) | Data access library |
+| [Net Core DI](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/dependency-injection?view=aspnetcore-2.0) | Integrated dependecy injection library |
+| [Xunit](https://xunit.github.io/) | Unit testing |
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Done!

File: Readme.md [hunk 8]
```diff
@@ -1,17 +1,166 @@
...
+| [SwaggerUI](https://swagger.io/swagger-ui/) | Ui that document and exposes the api endpoints |
```
[the remark this answers is not in the document]
Comment (Dongata) [reply Readme.md]: Done

File: Readme.md [hunk 9]
```diff
@@ -1,17 +1,62 @@
...
-    * Entity Framework(as DAL)
-    * Net Core DI
-    * Xunit
+# WebapiCore-seed [![Build Status](https://travis-ci.org/MakingSense/WebApiCore-Seed.svg?branch=master)](https://travis-ci.org/MakingSense/WebApiCore-Seed)
+
+## Prerequisites
+
+.NET Core SDK
+
+A text processor / gui
```
Comment (matiasbeckerle) [inline Readme.md]: Sorry, I did not express myself well. I was not referring to the markup bullet point but rather to the text processor/GUI/IDE requirement. I think it's pretty obvious you will need one to crack some code. But I do like having the prerequisites title with the SDK requirement.

**Pull request merged**